dcat/rdf/dcat3.ttl

```diff
@@ -1709,8 +1771,11 @@ dcat:themeTaxonomy
   rdfs:label "قائمة التصنيفات"@ar ;
   rdfs:label "テーマ"@ja ;
   rdfs:label "emnetaksonomi"@da ;
-  skos:altLabel "temataksonomi"@da ;
   rdfs:range rdfs:Resource ;
+  sdo:rangeIncludes owl:Ontology ;
```

Comment:
Should we replace `sdo:rangeIncludes` with [`dcam:rangeIncludes`](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#http://purl.org/dc/dcam/rangeIncludes)?